01_array_reverse/array_reverse.js

```diff
@@ -0,0 +1,15 @@
+'use strict';
+
+module.exports = exports = {};
+
+exports.reverseArray = (inputArr) => {
+  let reversedArr = [],
+    j = 0;
+  
+  for (let i = inputArr.length - 1; i >= 0; i--){

```

Comment:
An alternative way of writing the same functionality would be to declare `j` within the for loop condition (multiple for loop variable declarations):
```
for (let i = inputArr.length - 1, j = 0; i >= 0; i--, j++)